src/user/index.js

```diff
@@ -84,12 +84,16 @@ User.getUsers = async function (uids, uid) {
 	const userData = await User.getUsersWithFields(uids, [
 		'uid', 'username', 'userslug', 'picture', 'status',
 		'postcount', 'reputation', 'email:confirmed', 'lastonline',
-		'flags', 'banned', 'banned:expire', 'joindate',
+		'flags', 'banned', 'banned:expire', 'joindate', 'followerCount',
```

Comment:
The `followerCount` field doesn't follow the naming convention used for other fields and should be changed to `follower-count`